components/Activity/Agenda/Card.tsx

```diff
@@ -6,6 +6,18 @@ import { Card, Carousel } from 'react-bootstrap';
 import { blobURLOf } from '../../../models/Base';
 import { AgendaToolbar, AgendaToolbarProps } from './Toolbar';
 
+const renderCardImage = (file: TableCellAttachment) => {
+  return (
+    <Card.Img
+      key={file.attachmentToken}
+      className="m-auto object-fit-cover"
+      style={{ height: '25rem' }}
+      loading="lazy"
+      src={blobURLOf([file])}
+    />
+  );
+};
+
 @observer
 export class AgendaCard extends Component<AgendaToolbarProps> {
```

Comment:
你写的是普通函数，挂在类上才能叫“方法”：

```suggestion
@observer
export class AgendaCard extends Component<AgendaToolbarProps> {
  renderCardImage = (file: TableCellAttachment) => (
    <Card.Img
      key={file.attachmentToken}
      className="m-auto object-fit-cover"
      style={{ height: '25rem' }}
      loading="lazy"
      src={blobURLOf([file])}
    />
  );
```

Reply:
原来想提高可调用性